ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -1019,18 +1042,18 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
+				CURRENT_PROJECT_VERSION = 21;
```

Comment:
should we revert `CURRENT_PROJECT_VERSION` to use `"$(FLUTTER_BUILD_NUMBER)"` instead?